msbuild/Xamarin.Mac.Tasks.Core/Tasks/XamMacArch.cs

```diff
@@ -15,6 +15,7 @@ public enum XamMacArch
 	{
 		Default = 0,
 		i386 = 1,
-		x86_64 = 2
+		x86_64 = 2,
+		ARM64 = 4
```

Comment:
Minor, lets add a ',' at the end. Was not done, but will min the diff next time.

Reply:
Fixed at 1ed3508bc29a5e6cc385121a27544cd60e7d4bcc